### PR TITLE
[Templating] ease migration with template helpers

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
By adding them to 6.8 already and alias them to php template helpers

This allows Bundle providers to be compatible with 6.8 and 7.0 at the same time.

### CODE REVIEW

**Comment by brusch** [comment]
@dpfaffenbauer thanks a lot! can you please check this small Stan issue please: https://travis-ci.com/github/pimcore/pimcore/jobs/428053737#L1093 ?

**Comment by dpfaffenbauer** [comment]
one fixed, but I can't figure out this one:

```
------ -------------------------------------------- 
  Line   lib/Twig/Extension/Templating/HeadMeta.php  
 ------ -------------------------------------------- 
  314    Cannot access property $type on string.     
  314    Cannot access property $type on string.     
 ------ -------------------------------------------- 
```

This should be also happen to dev-master, but it doesn't?! (I copied the Twig files from master)

File: lib/Templating/Helper/Navigation/Exception/RendererNotFoundException.php [hunk 1]
```diff
@@ -17,10 +17,24 @@
 
 namespace Pimcore\Templating\Helper\Navigation\Exception;
 
-class RendererNotFoundException extends \InvalidArgumentException
-{
-    public static function create(string $name): self
+use Pimcore\Navigation\Renderer\RendererInterface;
```
Comment (dvesh3) [inline lib/Templating/Helper/Navigation/Exception/RendererNotFoundException.php:20]: required?

**Comment by dvesh3** [comment]
> one fixed, but I can't figure out this one:
> 
> ```
> ------ -------------------------------------------- 
>   Line   lib/Twig/Extension/Templating/HeadMeta.php  
>  ------ -------------------------------------------- 
>   314    Cannot access property $type on string.     
>   314    Cannot access property $type on string.     
>  ------ -------------------------------------------- 
> ```
> 
> This should be also happen to dev-master, but it doesn't?! (I copied the Twig files from master)

Added to Ignore list on master:
https://github.com/pimcore/pimcore/blob/a963d3d9899864748276627ec5ce58ee82c5ebdd/phpstan-baseline.neon#L1496

**Comment by dpfaffenbauer** [comment]
fixed

File: lib/Templating/Helper/Navigation/Exception/RendererNotFoundException.php (hunk 1 above)
Comment (brusch) [reply lib/Templating/Helper/Navigation/Exception/RendererNotFoundException.php:20]: @dpfaffenbauer This is useless, right?

Comment (dvesh3) [reply lib/Templating/Helper/Navigation/Exception/RendererNotFoundException.php:20]:
```suggestion
```

**Comment by brusch** [comment]
Changes base to `6.9`

**Comment by dpfaffenbauer** [comment]
@brusch can this get merged now?

**Comment by brusch** [comment]
@dpfaffenbauer after reviewing yes 😉

**Comment by brusch** [comment]
Thanks! 👍

**Comment by dpfaffenbauer** [comment]
thank you :)

**Pull request merged**